components/Poll/Poll.jsx

```diff
@@ -0,0 +1,342 @@
+          title: this.state.title,
+          multiSelect: this.state.multiSelect,
+          open: this.state.open,
+          options: poll.options,
+        });
+      })
+      .catch((error) => alert("Error adding poll option. Sorry about that!"));
+    } else {
+      var opts = this.state.options;
+      opts.push({
+        key: `${this.state.options.length}`,
+        text: optionText,
+        usersVoted: [],
+      });
+      this.props.savePoll({
+        title: this.state.title,
+        multiSelect: this.state.multiSelect,
+        open: this.state.open,
+        options: opts,
+      });
+    }
+  }
+
+  buildListItems = () => {
+    let items = this.state.options
+            .sort((a, b) => b.usersVoted.length - a.usersVoted.length)
+            .slice()
+            .map(item => {
+              if (item._id) {
+                item.key = item._id;
+              }
+              return item;
+            });
+    if (!!this.props.isAuthor || this.state.open) {
+      items.push({ key: 'add_option_item' });
+    }
+    return items;
+  }
+
+  renderListItem = ({ item }) => {
+    if (item.key === 'add_option_item') {
+      return (
+        <View style={styles.optionGroup}>
+          <TouchableOpacity style={styles.addContainingView} onPress={this.addOption}>
+            <Icon name={Platform.OS === 'ios' ? "ios-add" : "md-add"} style={styles.addView}/>
+          </TouchableOpacity>
+          <Textarea
+            placeholder="Add an option..."
+            style={styles.addOptionText}
+            onChangeText={this.newOptionUpdate}
+            value={this.state.newOption}
+          />
+        </View>
+      );
+    }
+    let selected = (this.userVoteIndex(item) !== -1);
+    return(
+  		<TouchableOpacity style={styles.fillWidth} onPress={this.toggleOption.bind(this, item)} onLongPress={this.confirmDelete.bind(this, item)}>
+        <View style={styles.optionView}>
+          {this.state.multiSelect ?
+            <CheckBox
+              style={styles.optionButton}
+              containerStyle={styles.optionButtonContainer}
+              checked={selected}
+              onIconPress={this.toggleOption.bind(this, item)}
+              onLongIconPress={this.confirmDelete.bind(this, item)}
+            /> :
+            <CheckBox
+              style={styles.optionButton}
+              containerStyle={styles.optionButtonContainer}
+              checkedIcon='dot-circle-o'
+              uncheckedIcon='circle-o'
+              checked={selected}
+              onIconPress={this.toggleOption.bind(this, item)}
+              onLongIconPress={this.confirmDelete.bind(this, item)}
+            />}
+  			  <Text style={styles.optionText}>{item.text}</Text>
+  			  <Text style={styles.optionCount}>{`${item.usersVoted.length}`}</Text>
+        </View>
+  		</TouchableOpacity>
+    );
+  }
+
+  render() {
+    // TODO fix: so many security vulnerabilities from unsanitized user input of option text and poll title
```

Comment:
Is this a fix for this PR, or just a TODO for a future one? Want to make sure this wasn't just forgotten about.

Reply:
Could we make sure to track TODOs left in the code? Especially if it's related to "security vulnerabilities"...

Reply:
This is probably going to a future PR. I was hoping to put it in this PR if I had time, but I don't think I have time. 
Is there a way to track TODOs?

Reply:
Put a task on Asana with the info for this todo

Reply:
Have you actually seen these issues? I'm not sure this is actually an issue.
The options/title are just strings, and rendered as such. I don't think it think they'd be able to modify the code in any fashion.

Worth verifying though, I guess